scripts/install_third_party_libs_test.py

```diff
@@ -438,10 +436,6 @@ def mock_copy_tree(src, dst):
                             with swap_is_dir, swap_mk_dir, swap_copy_tree:
                                 install_third_party_libs.main()
         self.assertEqual(check_function_calls, expected_check_function_calls)
-        with python_utils.open_file(temp_py_config_file, 'r') as f:
```

Comment:
Why are we dropping this and other assertions in this file?

Reply:
This is actually not needed anymore, previously we were getting rid of `ConverterMapping` from some libraries, but we don't need to do that in py3. I removed also another remaining part of the code.